internal/node_loader.js

```diff
@@ -102,6 +101,7 @@ module.constructor._resolveFilename =
   var resolveLocations = [
     request,
     resolveRunfiles(request),
+    resolveRunfiles('TEMPLATED_workspace_name', 'TEMPLATED_label_package', request),
```

Comment:
shouldn't need to add a fourth lookup location, at least without understanding why

Reply:
I needed the fourth lookup to be able to import `bundle.js` in the rollup jasmine tests.